### DISCUSSION TITLE
Bot crashed bro help me

### DISCUSSION THREAD

**Discussion by MR-JAGANMOHAN** [kalanakt/All-Url-Uploader#47] [Q&A]
Repo was deployed with all variables correct but it started and crashed continuesly and finally in crashed position 

[urlploder-logs-1660757377683.txt](https://github.com/kalanakt/All-Url-Uploader/files/9364957/urlploder-logs-1660757377683.txt)

**Comment by kalanakt** [comment]
### This error is fixed, you can update your forked repo and deploy again 📌
